src/pages/onboarding/Onboarding.vue.js

```diff
@@ -0,0 +1,97 @@
+import { swiper, swiperSlide } from 'vue-awesome-swiper'
+import router from '@/router'
+
+export default {
+  name: 'intro',
+  data () {
+    const vue = this
+    return {
+      colors: ['#C783FF', '#50E3C2', '#90DBFF', '#FF5537'],
+      swiperOption: {
+        notNextTick: true,
+        grabCursor: true,
+        setWrapperSize: true,
+        parallax: true,
+        pagination: '.swiper-pagination',
+        paginationClickable: true,
+        prevButton: '.swiper-button-prev',
+        nextButton: '.swiper-button-next',
+        mousewheelControl: true,
+        onTransitionStart(swiper) {
```

Comment:
Don't know how I could, this is the best solution I could come up with. Suggestions?